indexing-hadoop/src/main/java/org/apache/druid/indexer/DetermineHashedPartitionsJob.java

```diff
@@ -412,7 +412,7 @@ protected void reduce(
 
       try {
         HadoopDruidIndexerConfig.JSON_MAPPER
-            .writerWithType(Long.class)
+            .writerFor(Long.class)
```

Comment:
Lets put this in the forbidden api's ?

Reply:
Good point. I have added the check which would give the following error when the deprecated method is used:

```
[ERROR] Forbidden method invocation: com.fasterxml.jackson.databind.ObjectMapper#writerWithType(com.fasterxml.jackson.core.type.TypeReference) [Use ObjectMapper#writeFor instead]
[ERROR]   in org.apache.druid.server.coordinator.loading.HttpLoadQueuePeon (HttpLoadQueuePeon.java:135)
```